### PR TITLE
CVSL-1813 Making TIMED_OUT licences appear as NOT_STARTED for CA caselists

### PR DESCRIPTION
**Pull request by SebNegahbanMoJ**

### CODE REVIEW

File: server/services/caseloadService.ts [hunk 1]
```diff
@@ -104,7 +104,21 @@ export default class CaseloadService {
       })
 
     const [withLicence, pending] = await Promise.all([casesWithLicences, casesPendingLicence])
-    const casesWithComs = await this.mapResponsibleComsToCasesWithExclusions(withLicence.concat(pending))
+
+    const allCases = withLicence.concat(pending)
+    const allCasesWithStatusOverrides = allCases.map(c => {
+      const managedCase = c
```
Comment (SebNegahbanMoJ) [inline server/services/caseloadService.ts:110]: I find this declaration weird and unnecessary, but eslint didn't like reassignment of an existing param

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Comment by SebNegahbanMoJ** [comment]
Decided to go with an approach of overriding status closer to the view, rather than at the caseload level.

**Pull request closed without merging**